062-support-tiered-storage.md

```diff
@@ -0,0 +1,126 @@
+#  Support tiered storage natively in Strimzi
+
+## Current situation
+
+Kafka introduced tiered storage feature since 3.6. The feature is in early stage and it’s nice to support tiered storage configuration natively via Strimzi.
+
+## Motivation
+
+Supporting tiered storage via Strimzi allow Strimzi users to enable tiered storage natively with Strimzi config. By allowing to config custom configuration for `RemoteStorageManager`, users are able to adopt tiered storage feature on Strimzi easily.
```

Comment:
```suggestion
Supporting tiered storage in Strimzi would allow Strimzi users to enable tiered storage natively with Strimzi config. By allowing custom configuration of Kafka's `RemoteStorageManager` interface, Strimzi users would be able to easily adopt the tiered storage feature.
```